packages/grafana-ui/src/components/DataSourceSettings/AlertingSettings.tsx

```diff
@@ -1,13 +1,34 @@
-import { DataSourcePluginOptionsEditorProps } from '@grafana/data';
-import React from 'react';
+import { DataSourceInstanceSettings, DataSourceJsonData, DataSourcePluginOptionsEditorProps } from '@grafana/data';
+import React, { useMemo } from 'react';
 import { Switch } from '../Forms/Legacy/Switch/Switch';
+import { InlineField } from '../Forms/InlineField';
+import { InlineFieldRow } from '../Forms/InlineFieldRow';
+import { Select } from '../Select/Select';
 
-type Props<T> = Pick<DataSourcePluginOptionsEditorProps<T>, 'options' | 'onOptionsChange'>;
+interface Props<T> extends Pick<DataSourcePluginOptionsEditorProps<T>, 'options' | 'onOptionsChange'> {
+  alertmanagerDataSources: Array<DataSourceInstanceSettings<DataSourceJsonData>>;
```

Comment:
wondering, any reason to not just get available am datasources inside this component, so parent didn't have to worry abotu it?

Reply:
The `@grafana/runtime` cannot be imported in `@grafana/ui` so I'm not able to import the `config` from boot. Do you know of an alternative way?

Reply:
ah right, you already told me this, sorry